src/renderer/Components/Spaces/index.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * index.tsx
+ *
+ * Description:
+ *    Spaces Component.
+ *
+ */
+
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import List from './List';
+import Space from './Space';
+import './spaces.scss';
+
+const Spaces = () => {
+  return (
+    <Routes>
```

Comment:
Store all route attributes in an array of objects and use a map to populate all routes.